src/amr/data/particles/refine/split_1d.h

```diff
@@ -6,10 +7,16 @@ Splitting reference material can be found @
 #ifndef PHARE_SPLIT_1D_H
 #define PHARE_SPLIT_1D_H
 
+#include <array>                        // for array, array<>::value_type
```

Comment:
do we need those "for..xxx" comments? (I get they are automatic but...)

Reply:
need? no, I think it's useful to see at the top of the file tho what is used in the file

Reply:
depends how much stuff we use from a header, I wouldn't want a long list of stuff...
did IWYU do write that in the files? Did you run IWYU manually ? if yes, this kind of comments will never appear again ? Is that really consistent that we let them here and they don't appear anywhere else?

Reply:
it's pasted from https://github.com/PHAREHUB/PHARE/issues/299 

it's easily removable